src/ifcwrap-csharp/IfcOpenShell.Net/IfcOpenShell.Net/System.Numerics.Double/DoubleMatrix4x4.cs

```diff
@@ -0,0 +1,2374 @@
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+#if NET5_0_OR_GREATER
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+#endif
+
+namespace System.Numerics.Double
```

Comment:
Add a comment that this is copy pasted from System.Numerics on this (and all related) files.

Alternatively, see if you can get rid of them